src/main/webapp/app/course/competencies/competency-accordion/competency-accordion.component.html

```diff
@@ -0,0 +1,80 @@
+<div>
+    <button
+        [id]="'competency-accordion-' + competency.id"
+        class="competency-accordion-header row align-items-center justify-content-between mt-2 position-relative flex-wrap"
+        (click)="toggle()"
+    >
+        <div class="col-auto d-md-block my-2">
+            <h3 class="fw-medium">
+                <fa-icon
+                    [icon]="getIcon(competency.taxonomy)"
+                    [fixedWidth]="true"
+                    [ngbTooltip]="'artemisApp.competency.taxonomies.' + (competency.taxonomy ?? 'none') | artemisTranslate"
+                    container="body"
+                />
+            </h3>
+        </div>
+        <div class="col-sm col py-2">
+            <h3>{{ competency.title }}</h3>
+        </div>
+        @if (!open) {
+            <div class="col-sm col py-2" style="max-width: 200px" [class.header-tooltip-open]="open">
+                <div class="row mb-2">
+                    <div class="col-auto d-md-block">
+                        <fa-icon [icon]="faList" [ngbTooltip]="'artemisApp.studentAnalyticsDashboard.competencyAccordion.exercises' | artemisTranslate" />
+                    </div>
+                    <div class="col">
+                        <jhi-student-analytics-dashboard-progress-bar [size]="'small'" [maxValue]="100" [currentValue]="exercisesProgress" [inline]="true" />
+                    </div>
+                </div>
+                <div class="row align-items-center justify-content-center">
+                    <div class="col-auto d-md-block">
+                        <fa-icon [icon]="faPdf" [ngbTooltip]="'artemisApp.studentAnalyticsDashboard.competencyAccordion.lectures' | artemisTranslate" />
+                    </div>
+                    <div class="col">
+                        <jhi-student-analytics-dashboard-progress-bar [size]="'small'" [maxValue]="100" [currentValue]="lectureUnitsProgress" [inline]="true" />
```

Comment:
The progress bars are also very buggy. They seem to update very inconsistently. I also can't get the exercise one to fill. Can you test this more to see if the state updates are correct?

Reply:
Actually everything has very strange state updates. Also the competency rings.

Reply:
I will look into this, but competency rings should be consistent with the one in the competency detail page, since both use essentially the same component

Reply:
Do you have an example regarding the exercises that you can provide to me? If the exercise is regarded as not completed, then the progress bar also won't fill.

Reply:
I created an exercise with an competency and even lectures and completed them. Afterwards I checked the dashboard and the progress was still empty. I clicked on the competency rings and then it updated for the competency detail page. I went back and it finally was updated there.